ledger/ledger_test.go

```diff
@@ -1554,3 +1576,125 @@ func TestListAssetsAndApplications(t *testing.T) {
 	}
 	require.Equal(t, appCount, len(results))
 }
+
+func TestLedgerMemoryLeak(t *testing.T) {
+	dbName := fmt.Sprintf("%s.%d", t.Name(), crypto.RandUint64())
+	genesisInitState, initKeys := testGenerateInitState(t, protocol.ConsensusCurrentVersion)
+	const inMem = true
```

Comment:
I think that if you're trying to have a memory leak test, you should avoid memory file databases..